models/import_ufo.py

```diff
@@ -349,6 +349,7 @@ def import_full_model(model_path, decay=False, prefix=''):
                 raise UFOImportError("%s directory is not a valid UFO model: \n %s is missing" % \
                                                          (model_path, filename))
         files_list.append(filepath)
+    files_list.append(os.path.join(MG5DIR, 'models', 'import_ufo.py')) # see mg5amcnlo/mg5amcnlo#89
```

Comment:
```suggestion
    files_list.append(__file__) # ensure that modification to this file changes the cache
```
This sounds even more secure to me. __file__ always point towards the current executed, but if you compile the code where it can be problematic, but in that case we are in trouble with both methods

Reply:
Also done now and tested, thanks.